plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDeviceManager.py

```diff
@@ -230,7 +230,8 @@ def _onDevicesDiscovered(self, clusters: List[CloudClusterResponse]) -> None:
             ),
             progress = 0,
             lifetime = 0,
-            image_source = image_path
+            image_source = image_path,
+            message_type = Message.MessageType.POSITIVE
```

Comment:
Not entirely sure if that this should be positive, but i can see a bit of the logic. Maybe we should discuss this with Lorenzo?